refresh.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
perhaps there's a better name for this file. Maybe `install.sh` or `setup_env.sh` or something